apps/web/src/global.css

```
@@ -0,0 +1,139 @@
+[data-bs-theme="light"] {
+  --fasti-surface-archive: #f2efe6;
+  --fasti-surface-paper: #fffdf8;
+  --fasti-surface-night: #11110f;
+  --fasti-text-primary: #181716;
+  --fasti-text-muted: #625e56;
+  --fasti-brand-mark: #8b2e2a;
+  --fasti-brand-gold: #d4af37;
+  --fasti-action-primary: #066fd1;
+  --fasti-state-verified: #2fb344;
+  --fasti-state-attention: #f59f00;
+  --fasti-font-display: "Newsreader", Georgia, serif;
```

Stylelint (17.14.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)